nix/default.nix

```diff
@@ -14,6 +19,35 @@ let
   python = import ./requirements.nix { inherit pkgs; };
   version = builtins.replaceStrings ["\n"] [""]
     (builtins.readFile (toString ../version.txt));
+  scriptWorkerConfig = pkgs.writeTextDir "scriptworker.yaml" (builtins.readFile ./scriptworker.yaml);
+  workerConfigs = ./configs;
+  cmd = pkgs.writeScriptBin "generateconfig" ''
+      #!${pkgs.bash}/bin/bash
+      set -ex
+      mkdir -p -m 700 $CONFIGDIR
+      # TaskCluster doesn't accept WORKER_ID longer than 22 chars
+      export WORKER_ID=''${HOSTNAME:0:22}
+      export TASK_SCRIPT_CONFIG="$CONFIGDIR/worker.json"
+      $CONFIGLOADER $SCRIPTWORKER_CONFIG_TEMPLATE $CONFIGDIR/scriptworker.json
+      $CONFIGLOADER $TASK_SCRIPT_CONFIG_TEMPLATE_DIR/$APP_CHANNEL/worker.json $TASK_SCRIPT_CONFIG
+      $GIT_PATH config --global gpg.program $GPG_PATH
+      $GIT_PATH clone https://github.com/mozilla-releng/cot-gpg-keys.git /app/gpg_key_repo
+      echo $PUBLIC_KEY | base64 -d > /app/pubkey
+      echo $PRIVATE_KEY | base64 -d > /app/privkey
+      $(dirname $SCRIPTWORKER)/rebuild_gpg_homedirs $CONFIGDIR/scriptworker.json
+      $SCRIPTWORKER $CONFIGDIR/scriptworker.json
+    '';
+    # TODO: should this be linked to a stable location, /check.sh?
+    healthcheck = pkgs.writeScriptBin "healthcheck" ''
+      #!${pkgs.bash}/bin/bash
+      created=$(date -r /app/logs/worker.log +%s)
```

Comment:
we should use `/tmp` as place where to log

Reply:
Easy!